x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_detail_panel/data_stream_detail_panel.tsx

```diff
@@ -74,7 +99,7 @@ export const DataStreamDetailPanel: React.FunctionComponent<Props> = ({
     >
       <EuiFlyoutHeader>
         <EuiTitle size="m">
-          <h2 id="dataStreamDetailPanelTitle" data-test-subj="title">
+          <h2 id="dataStreamDetailPanelTitle" data-test-subj="dataStreamDetailPanelTitle">
```

Comment:
Good point! I prefer to use a selector that references a specific element instead of depending upon context, so I'm going to leave this as-is for now.